### PR TITLE
Improve sizing and handling of news slider and stairs

### PR DESCRIPTION
**Pull request by snaegle**
Related to: AAC-721, AAC-335, and AAC-729

### CODE REVIEW

File: Resources/Private/Scss/_news.scss [hunk 1]
```diff
@@ -118,4 +121,17 @@
             }
         }
     }
+
+    .teaser-text {
+        padding: 0;
+        margin: 0 !important;
```
Comment (nilswindisch) [inline Resources/Private/Scss/_news.scss]: !important should be considered a no-go

File: Resources/Private/Scss/blocks/news-slider.scss [hunk 2]
```diff
@@ -14,15 +18,19 @@
...
         @include media-m {
             @include span-m-5;
+            width: width(m-5);
+            margin: $magic-m/2 $magic-m $magic-m/2 0;
         }
 
         @include media-s {
             @include span-s-6;
+            width: width(s-6);
+            margin: $magic-s/2;
```
Comment (nilswindisch) [inline Resources/Private/Scss/blocks/news-slider.scss]: margin-top is considered bad practice. See AAC-716

**Comment by nilswindisch** [comment]
1. When the image is on the right, text on the left should be right aligned
2. padding-top missing, the top element touches the menu bar.

**Review by nilswindisch — CHANGES_REQUESTED** [review]
see various comments

**Comment by nilswindisch** [comment]
screenshot 2017-10-04 15 43 11
I suggest to remove the margin/padding right of the last category item when aligned to the right.

**Comment by nilswindisch** [comment]
In contrast to "Amerian Studies" "Great Britain" and "History" are not clickable, on purpose?

**Comment by snaegle** [comment]
"Great Britain" and "History" are not configured, yet

**Review by nilswindisch — APPROVED** [review]

**Pull request merged**